test/transaction.test.js

```diff
@@ -0,0 +1,235 @@
+// Copyright IBM Corp. 2013,2019. All Rights Reserved.
+// Node module: loopback-connector-mongodb
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+const should = require('./init.js');
+const chaiAsPromised = require('chai-as-promised');
```

Comment:
You may remove this line, chai-as-promised is not needed. I copied your PR and all tests passed.